src/components/Trade/TradeInput.tsx

```diff
@@ -409,9 +444,17 @@ export const TradeInput = () => {
           assetSymbol: sellAsset?.symbol ?? translate('trade.errors.sellAssetStartSentence'),
         },
       ]
-    if (!walletSupportsBuyAssetChain)
+
+    if (manualAddressEntryErrors.input?.message && !walletSupportsBuyAssetChain) {
+      return manualAddressEntryErrors.input?.message.toString()
+    }
+
+    if (
+      !walletSupportsBuyAssetChain &&
+      (!receiveAddress || manualAddressEntryErrors?.input?.type?.toString() === 'required')
```

Comment:
Looks like this is effectively the same check? Though there's no such thing as "too much safety" here, just want to make sure this was intentional

Reply:
From memory `manualAddressEntryErrors?.input?.type?.toString() === 'required')` caught an edge case, though I'll need to check when THORChain comes back online so I can play with this functionality properly again.

Reply:
Ok yes, it catches an edge case when the full address is deleted after being validated, the form won't re-fire until it receives another value. I think this is because our parent form mode is set to `mode: 'onChange'`, which means the validation trigger won't fire on empty value.

As touches this mode affects all child forms in Trade Routes, I'm hesitant to touch it in this PR.

Reply:
Yeah let's tackle it in a follow-up 🙏🏽 if the intent is indeed to validate initial empty value, `trigger()` within a no-deps `useEffect()` (effectively on mount) should do the same